### PR TITLE
Work on basic psql backup to gcp

### PR DESCRIPTION
**Pull request by domudall**
Basic validation, could probably do with some additional work but currently performs correctly.

### CODE REVIEW

File: psql-gcp-backup.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (lbn) [inline psql-gcp-backup.sh]:
```
#!/usr/bin/env bash
```
recommended

File: psql-gcp-backup.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+if [ "$2" == "" ]; then
+    echo "Bucket name required"
+    exit 1
+fi
+
+DATE=$(date +%Y%m%d_%H%M)
+DB_NAME=$1
+BUCKET=$2
+
+if [[ `psql -U postgres -lqt | cut -d \| -f 1 | grep -qw $DB_NAME` != "0" ]]
```
Comment (lbn) [inline psql-gcp-backup.sh]:
```
$()
```
recommended over ``` `` ```

File: psql-gcp-backup.sh (hunk 1 above)
Comment (domudall) [inline psql-gcp-backup.sh]: Fixed

File: psql-gcp-backup.sh (hunk 2 above)
Comment (domudall) [inline psql-gcp-backup.sh]: Fixed

**Pull request merged**